src/panels/lovelace/cards/hui-stack-card.ts

```diff
@@ -0,0 +1,63 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import createCardElement from "../common/create-card-element.js";
+
+import { LovelaceCard, LovelaceConfig } from "../types";
+import { HomeAssistant } from "../../../types";
+
+interface Config extends LovelaceConfig {
+    cards: LovelaceConfig[];
+}
+
+export default class HuiStackCard extends LitElement implements LovelaceCard {
+    protected config?: Config;
+    private _hass?: HomeAssistant;
+
+    static get properties() {
+        return {
+            config: {},
+        };
+    }
+
+    set hass(hass: HomeAssistant) {
+        this._hass = hass;
+        for (const el of this.shadowRoot!.querySelectorAll("#root > *")) {
+            const element = el as LovelaceCard;
+            element.hass = this._hass;
+        }
+    }
+
+    public getCardSize() {
+        return 1;
+    }
+
+    public setConfig(config: Config) {
+        if (!config || !config.cards || !Array.isArray(config.cards)) {
+            throw new Error("Card config incorrect");
+        }
+        this.config = config;
+    }
+
+    protected render() {
+        if (!this.config) {
+            return html``;
+        }
+
+        return html`
+      ${this.renderStyle()}
+      <div id="root">
+        ${this.config.cards.map((card) => this.createCardElement(card))}
+      </div>
+    `;
+    }
+
+    protected renderStyle() {
```

Comment:
You don't need this function if you aren't using it

Reply:
It is used above on line 47 and is overridden by `HuiHorizontalStackCard` which extends this class

Reply:
Perfect! As @thomasloven [mentioned](https://github.com/home-assistant/home-assistant-polymer/pull/1853#issuecomment-432943050) might be a good idea for card with image backgrounds as well